scripts/build-linux.sh

```diff
@@ -258,7 +265,11 @@ if [ ${CPU} = vexriscv ] && [ ${BUILD_BUILDROOT:-0} = 1 ]; then
 		dtc -I dts -O dtb -o $TARGET_LINUX_BUILD_DIR/rv32.dtb $TARGET_LINUX_BUILD_DIR/rv32.dts
 
 		cd $BD_SRC
-		make BR2_EXTERNAL=$LLV_SRC/buildroot/ litex_vexriscv_defconfig
+                if [ "x$BR2_EXTERNAL" = "x" ]; then
```

Comment:
I'm a bit suspicious of this if statement. Can you check it @mateusz-holenko?

Reply:
I checked it locally and it seems that having a `BR2_EXTERNAL` variable set in the shell is not enough. I had to export it using `export BR2_EXTERNAL` in order for `make` to see/use it.

This test do not check if the variable is exported, so I guess it might give wrong results.

Reply:
Good point, I have only tested with BR2_EXTERNAL exported, I'll fix it.

Reply:
The current solution should work, but wouldn't it be simpler to use:
```
make BR2_EXTERNAL=${BR2_EXTERNAL:-$LLV_SRC/buildroot} litex_vexriscv_defconfig
```
?

Reply:
Yes, but I think that has the same issue that you commented on yesterday.  Namely it doesn't test if BR2_EXTERNAL has been exported or simply set.  Personally I'm not too worried about that.

Reply:
I mean, just `make ...` without any `ifs` around it.

With this apporach you don't need to export `BR2_EXTERNAL` in your shell anymore. AFAIK `make X=1` is not the same as `X=1 make`.

I tested it locally and it seemed to work.

Reply:
Done!

Reply:
Looks good.